pkg/os/shell/shell_windows_test.go

```diff
@@ -33,6 +35,8 @@ func TestGetNameAndItsPpidOfParent(t *testing.T) {
 }
 
 func TestGetNameAndItsPpidOfGrandParent(t *testing.T) {
+	t.Skip("not working on AppVeyor")
```

Comment:
Why would we have those tests if we are going to skip those, I think this will skipped on any windows system. I think it is failing because we are asserting powershell, even in appveyour it execute on cmd?

Reply:
I tried to use ps on AppVeyor but because of we use `make test`, there is a sh.exe also in the party... :/